client/src/resources/IdGenerator.js

```diff
@@ -8,7 +8,20 @@ export default class IdGenerator {
   }
 
   getId() {
-    return Date.now() + "-" + (Math.random() * 10 ** 9).toFixed(0);
+    return (
```

Comment:
Could you explain what the reason for this change is? Imo we should not change code if there is no good reason for it. The previous type of id with the timestamp and some random number workes fine as far as I see.